gradle/scripts/lib/java.gradle

```diff
@@ -92,7 +92,18 @@ configure(projectsWithFlags('java')) {
         testLogging.exceptionFormat = 'full'
 
         // Use JUnit platform.
-        useJUnitPlatform()
+        def flakyTest = rootProject.findProperty('flakyTest')
+        if (flakyTest == 'true') {
+            useJUnitPlatform {
+                includeTags 'FLAKY_TEST'
+            }
+        } else if (flakyTest == 'false') {
+            useJUnitPlatform {
+                excludeTags 'FLAKY_TEST'
+            }
+        } else {
+            useJUnitPlatform()
```

Comment:
How about raising an exception if `flakyTest` is not `null`?

Reply:
Sounds nice. 👍